src/tests/e2etest/E2ETest.cs

```diff
@@ -46,6 +46,14 @@ public void SetTwinTimeoutSeconds(int seconds)
             twinTimeoutSeconds = seconds;
         }
 
+        public void CheckModuleConnection()

```

Comment:
This method will break the test on assertion failure as such, i would recommend renaming the method to `AssertModuleConnected` or something similar.